tests/test_common.py

```diff
@@ -82,7 +82,6 @@ def libftdi_version(self):
 
 
 # importing this _does_ things...
-pylibftdi.driver.Driver = MockDriver
```

Comment:
Could you clarify the purpose of the reassigned Driver class? I was able to omit it and still pass the unit tests, but I'm unsure if it affects anything else. Left in, the reassignment breaks the unit tests for `test_driver.py` since MockDriver has no attribute `_lib_search`.